src/redux/local/localSlice.ts

```diff
@@ -27,11 +33,40 @@ export const localSlice = createSlice({
     setSceneTreeSelectedNode: (state, action: PayloadAction<Uri | null>) => {
       state.sceneTree.currentlySelectedNode = action.payload;
       return state;
+    },
+    setVisibleMenuItem: (
+      state,
+      action: PayloadAction<{ id: string; visible: boolean }>
+    ) => {
+      const item = state.taskbarItems.find((item) => item.id === action.payload.id);
+      if (item) {
+        item.visible = action.payload.visible;
+      }
+      return state;
+    },
+    setEnabledMenuItem: (
```

Comment:
tiny thing.  Maybe this should be `setMenuItemEnabled` and `setMenuItemVisible`?

`